### PR TITLE
Rename decision-making bodies to councils

### PR DESCRIPTION
**Pull request by jasongrout**
Thanks for participating in Jupyter governance ✨
Below are a few guidelines for making your proposed change as efficient and productive
as possible.

## Questions to answer ❓

Please answer the following questions along with your proposed change:

**Background or context to help others understand the change.**

Some of us find the term "decision-making body" an awkward term.

**A brief summary of the change.**

This change is based on discussion with several people. I tried to preserve the generic term "decision-making body" for the overall concept, while calling the subproject bodies "councils".

**What is the reason for this change?**

**Alternatives to making this change and other considerations.**

## The process ❗

The process for changing the governance pages is as follows:

* Open a pull request **in draft state**. This triggers a discussion and iteration phase
  for your proposed changes.
* When you believe enough discussion has happened,
  **move the pull request to an active state**. This triggers a vote.
* During the voting phase, no substantive changes may be made to the pull request.
* The Steering Council will vote, and at the end of voting the pull request is merged or closed.

The discussion phase is meant to gather input and multiple perspectives from the community.
Make sure that the community has had an opportunity to weigh in on
the change **before calling a vote**. A good rule of thumb is to ask several steering council
members if they believe that it is time for a vote, and to let at least one person review
the pull request for structural quality and typos.

---

This vote closes on 10 May 2022 (anywhere on Earth) or earlier if 14 or more members of the Steering Council have voted.

> All votes are limited in time to 4 weeks after the voting phase begins. At the end of 4 weeks, the proposal passes if 2/3 of the votes are in favor (fractions of a vote rounded up to the nearest integer); otherwise the proposal is rejected and the PR is closed. Prior to the four-week limit, if at least 80% of the Steering Council has voted and 2/3 of the votes are in favor (fractions of a vote rounded up to the nearest integer), the proposal passes.

(cf. [governance.md](https://github.com/jupyter/governance/blob/master/governance.md#changing-the-governance-documents))

---

## Votes

@afshin
- [x] YES
- [ ] NO

@blink1073
- [x] YES
- [ ] NO

@Carreau
- [x] YES
- [ ] NO

@damianavila
- [x] YES
- [ ] NO

@ellisonbg
- [x] YES
- [ ] NO

@fperez
- [x] YES
- [ ] NO

@ivanov
- [ ] YES
- [ ] NO

@jasongrout
- [x] YES
- [ ] NO

@jhamrick
- [ ] YES
- [ ] NO

@minrk
- [x] YES
- [ ] NO

@mpacer
- [x] YES
- [ ] NO

@parente
- [ ] YES
- [ ] NO

@rgbkrk
- [x] YES
- [ ] NO

@Ruv7
- [x] YES
- [ ] NO

@SylvainCorlay
- [ ] YES
- [ ] NO

@takluyver
- [ ] YES
- [ ] NO

@willingc
- [x] YES
- [ ] NO

### CODE REVIEW

**Comment by jasongrout** [comment]
CC @fperez

**Review by fperez — APPROVED** [review]
Excellent, thanks for doing this @jasongrout!

File: bootstrapping_decision_making.md [hunk 1]
```diff
@@ -1,34 +1,34 @@
-# Bootstrapping Decision-Making Bodies
+# Bootstrapping Subproject Councils
```
Comment (lresende) [inline bootstrapping_decision_making.md:1]: How about Project Management Council?

**Comment by ellisonbg** [comment]
Thanks Jason, I think this looks good.

File: bootstrapping_decision_making.md (hunk 1 above)
Comment (ellisonbg) [reply bootstrapping_decision_making.md:1]: FWIW, in the new governance model the "Project" is Jupyter and the different software "things" are Subprojects.

**Review by afshin — APPROVED** [review]

**Comment by Carreau** [comment]
No objections.

**Review by minrk — APPROVED** [review]

**Comment by jasongrout** [comment]
Following the discussion in the governance meeting today, I rebased the original commit in this PR on top of current HEAD, as well as added a new commit expanding our use of the word "council" to any Jupyter decision-making body.

If there is general agreement about this expanded language, and no objections by early next week, I'll move this to the voting stage.

Flagging this additional commit for those that already voted: @afshin, @damianavila, @minrk, @Carreau, @fperez, @ellisonbg

**Review by Carreau — APPROVED** [review]

**Review by minrk — APPROVED** [review]

File: bootstrapping_decision_making.md [hunk 2]
```diff
@@ -1,34 +1,34 @@
...
 
-As the new Jupyter governance model is approved and implemented, we are moving from an informal Subproject governance model to a formal model. As part of this process, each Jupyter Subproject will need to establish a formal decision-making body, including deciding its size and who its members are. There are a number of challenges associated with this:
+As the new Jupyter governance model is approved and implemented, we are moving from an informal Subproject governance model to a formal model. As part of this process, each Jupyter Subproject will need to establish a formal Subproject council as its decision-making body, including deciding its size and who its members are. There are a number of challenges associated with this:
```
Comment (ellisonbg) [inline bootstrapping_decision_making.md:3]: We have been trying to be consistent in capitalizing the formal nouns in the governance model. Maybe council should be capitalized to follow the pattern?

File: decision_making.md [hunk 3]
```diff
@@ -1,6 +1,6 @@
 # Decision-Making Guide
 
-This document describes how Jupyter’s governing bodies make decisions. Because individuals often work across multiple decision-making groups, all Jupyter governing bodies are expected to follow this approach.
+This document describes how Jupyter’s governing bodies (generally called a "council") make decisions. Because individuals often work across multiple decision-making groups, all Jupyter governing bodies are expected to follow this approach.
```
Comment (ellisonbg) [inline decision_making.md:3]: Use plural "councils" to match "bodies"

**Review by ellisonbg — CHANGES_REQUESTED** [review]
Minor comments.

File: bootstrapping_decision_making.md (hunk 2 above)
Comment (jasongrout) [reply bootstrapping_decision_making.md]: I audited capitalization here and other places in a new commit I just pushed.

**Comment by jasongrout** [comment]
For convenience, I added the voting form in the description, and checked yes for @Carreau and @minrk since they reaffirmed their positive vote in the last few days.

**Review by damianavila — APPROVED** [review]

**Review by willingc — APPROVED** [review]

**Review by fperez — APPROVED** [review]

**Comment by jasongrout** [comment]
Just a soft reminder ping to vote to @ivanov, @jhamrick, @mpacer, @parente, @rgbkrk, @Ruv7, @SylvainCorlay, @takluyver

**Review by rgbkrk — APPROVED** [review]

**Comment by jasongrout** [comment]
We now have 12 positive votes, no negatives, and the 4 weeks has passed, so this PR passes.

**Pull request merged**